src/hooks/useSettings.ts

```diff
@@ -27,13 +27,19 @@ const initialState = mergeDeepRight(
   window.leckerliSettings ?? {}
 );
 
+// Build the expiration date
+const expirationDate = new Date();
+expirationDate.setMonth((new Date().getMonth() % 12) + 1);
```

Comment:
Can you do something like that : `expirationDate.setMonth(expirationDate.getMonth() + 1);`  ?

Reply:
if I do that, then we have an issue in December ;) @Gurakh

Reply:
The Date object should take care of that automatically : 

https://ramdajs.com/repl/?v=0.29.0#?const%20date%20%3D%20new%20Date%282022%2C%2011%2C%2010%29%3B%0A%0Adate.setMonth%28date.getMonth%28%29%20%2B%201%29%3B%0A%0Aconsole.log%28date%29%3B